interface/forms/newpatient/common.php

```diff
@@ -76,444 +76,459 @@ function sensitivity_compare($a, $b)
-        $issuesauth = false;
-        break;
-    }
+if ($GLOBALS['enable_help'] == 1) {
+    $help_icon = '<a class="pull-right oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#676666" title="' . xl("Click to view Help") . '"><i class="fa fa-question-circle" aria-hidden="true"></i></a>';
+} elseif ($GLOBALS['enable_help'] == 2) {
+    $help_icon = '<a class="pull-right oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#DCD6D0 !Important" title="' . xl("Enable help in Administration > Globals > Features > Enable Help Modal") . '"><i class="fa fa-question-circle" aria-hidden="true"></i></a>';
+} elseif ($GLOBALS['enable_help'] == 0) {
+     $help_icon = '';
 }
-
-if ($issuesauth) {
 ?>
-  <div style='float:left'>
-<?php echo xlt('Issues (Injuries/Medical/Allergy)'); ?>
-  </div>
-  <div style='float:left;margin-left:8px;margin-top:-3px'>
-    <?php if (acl_check('patients', 'med', '', 'write')) { ?>
-       <a href="../../patient_file/summary/add_edit_issue.php" class="css_button_small link_submit enc_issue"
-        onclick="top.restoreSession()"><span><?php echo xlt('Add'); ?></span></a>
-        <?php } ?>
-  </div>
-<?php } ?>
-
-  </td>
- </tr>
-
- <tr>
-  <td class='text' valign='top'>
-   <textarea name='reason' cols='40' rows='12' wrap='virtual' style='width:96%'
-    ><?php echo $viewmode ? text($result['reason']) : text($GLOBALS['default_chief_complaint']); ?></textarea>
-  </td>
-  <td class='text' valign='top'>
-
-<?php if ($issuesauth) { ?>
-   <select class="form-control" multiple name='issues[]' size='8' style='width:100%'
-    title='<?php echo xla('Hold down [Ctrl] for multiple selections or to unselect'); ?>'>
-<?php
-while ($irow = sqlFetchArray($ires)) {
-    $list_id = $irow['id'];
-    $tcode = $irow['type'];
-    if ($ISSUE_TYPES[$tcode]) {
-        $tcode = $ISSUE_TYPES[$tcode][2];
-    }
-
-    echo "    <option value='" . attr($list_id) . "'";
-    if ($viewmode) {
-        $perow = sqlQuery("SELECT count(*) AS count FROM issue_encounter WHERE " .
-        "pid = ? AND encounter = ? AND list_id = ?", array($pid,$encounter,$list_id));
-        if ($perow['count']) {
-            echo " selected";
-        }
-    } else {
-        // For new encounters the invoker may pass an issue ID.
-        if (!empty($_REQUEST['issue']) && $_REQUEST['issue'] == $list_id) {
-            echo " selected";
-        }
+</head>
+<body class="body_top" <?php echo $body_javascript;?>>
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-12">
+                <!-- Required for the popup date selectors -->
+                <div id="overDiv" style="position:absolute; visibility:hidden; z-index:1000;"></div>
+                <div class="">
+                    <div class="page-header">
+                        <h2><?php echo xlt($heading_caption); ?><?php echo $help_icon; ?></h2>
```

Comment:
As suggested above, do the xl() above and then just do text() here

Reply:
will fix